### PR TITLE
prow/plugins/size,lgtm: expose config through PluginHelp

### PR DESCRIPTION
**Pull request by ibrasho**
The following plugins didn't expose their current config through `PluginHelp`:
* `lgtm`
* `size`

This PR update the plugins to expose this information.

Fixes: #10714

### CODE REVIEW

File: prow/plugins/lgtm/lgtm.go [hunk 1]
```diff
@@ -60,9 +60,37 @@ func init() {
 }
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
-	// The Config field is omitted because this plugin is not configurable.
+	willNot := func(b bool) string {
+		if b {
+			return "will"
+		}
+		return "will not"
```
Comment (taragu) [inline prow/plugins/lgtm/lgtm.go]: To make it consistent, please remove the extra whitespace in `prow/plugins/approve/approve.go` `return "do not "` and `return "will not "`

File: prow/plugins/size/size.go [hunk 2]
```diff
@@ -50,15 +50,25 @@ func init() {
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
 	// Only the Description field is specified because this plugin is not triggered with commands and is not configurable.
```
Comment (taragu) [inline prow/plugins/size/size.go]: Please update this comment

**Comment by fejta** [comment]
/lgtm
thanks

File: prow/plugins/lgtm/lgtm.go [hunk 3]
```diff
@@ -60,9 +60,37 @@ func init() {
...
+	}
+	lgtmConfig := map[string]string{}
+	for _, repo := range enabledRepos {
+		parts := strings.Split(repo, "/")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("invalid repo in enabledRepos: %q", repo)
+		}
+		opts := optionsForRepo(config, parts[0], parts[1])
+		lgtmConfig[repo] = fmt.Sprintf("Github reviews %s act as adding or removing the LGTM label.<br>"+
+			"Tree hash %s be stored in a comment to detect squashed commits before removing LGTM label.<br>"+
```
Comment (stevekuznetsov) [inline prow/plugins/lgtm/lgtm.go]: `s/hash/hashes/`

File: prow/plugins/lgtm/lgtm.go [hunk 4]
```diff
@@ -60,9 +60,37 @@ func init() {
...
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
-	// The Config field is omitted because this plugin is not configurable.
+	willNot := func(b bool) string {
+		if b {
+			return "will"
+		}
+		return "will not"
+	}
+	definedNot := func(s string) string {
+		if s != "" {
```
Comment (stevekuznetsov) [inline prow/plugins/lgtm/lgtm.go]: Shouldn't this be `==`?

File: prow/plugins/lgtm/lgtm.go [hunk 5]
```diff
@@ -60,9 +60,37 @@ func init() {
...
+			return "(not defined)"
+		}
+		return s
+	}
+	lgtmConfig := map[string]string{}
+	for _, repo := range enabledRepos {
+		parts := strings.Split(repo, "/")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("invalid repo in enabledRepos: %q", repo)
```
Comment (stevekuznetsov) [inline prow/plugins/lgtm/lgtm.go]: Can't we assume this?

File: prow/plugins/approve/approve.go [hunk 6]
```diff
@@ -115,7 +115,15 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 			return nil, fmt.Errorf("invalid repo in enabledRepos: %q", repo)
 		}
 		opts := optionsForRepo(config, parts[0], parts[1])
-		approveConfig[repo] = fmt.Sprintf("Pull requests %s require an associated issue.<br>Pull request authors %s implicitly approve their own PRs.<br>The /lgtm [cancel] command(s) %s act as approval.<br>A GitHub approved or changes requested review %s act as approval or cancel respectively.", doNot(opts.IssueRequired), doNot(opts.HasSelfApproval()), willNot(opts.LgtmActsAsApprove), willNot(opts.ConsiderReviewState()))
+		approveConfig[repo] = fmt.Sprintf("Pull requests %s require an associated issue.<br>"+
+			"Pull request authors %s implicitly approve their own PRs.<br>"+
+			"The /lgtm [cancel] command(s) %s act as approval.<br>"+
+			"A GitHub approved or changes requested review %s act as approval or cancel respectively.",
```
Comment (stevekuznetsov) [inline prow/plugins/approve/approve.go]: Maybe

```
GitHub reviews that approve changes or request changes %s...
```

**Comment by matthyx** [comment]
@ibrasho I'm sorry, I didn't pay attention and committed a change to size.go for the same issue...
If you want, you can still fix this PR and commit your work for lgtm, or let me know if I should handle it as well.

**Comment by ibrasho** [comment]
No worries. Go ahead with `lgtm` as well.

**Pull request closed without merging**